commons/src/main/java/org/openmrs/eip/app/management/config/ManagementDataSourceConfig.java

```diff
@@ -3,9 +3,10 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import javax.persistence.EntityManagerFactory;
 import javax.sql.DataSource;
 
+import jakarta.persistence.EntityManagerFactory;
```

Comment:
Nit: It seems to me that whatever formatting is being used here should regard `jakarta.` as similar to `javax.`, since it largely replaces `javax.` for JEE-related factors.

In other words, I think things would look a little neater if we had a rule that set things up like:

```
import java.*

import javax.*

import jakarta.*

import *
```

Reply:
In that regard, we need to setup auto-formatting for this repository as there isn't one, [OpenMRS Formatter](https://github.com/openmrs/openmrs-core/blob/master/tools/src/main/resources/eclipse/OpenMRSFormatter.xml) should be ok.

Reply:
PR https://github.com/openmrs/openmrs-eip/pull/44 to sort out imports and formatting.